imaging_install.sh

```diff
@@ -218,20 +218,28 @@ echo
 ################################################################################################
 #####################################DICOM TOOLKIT##############################################
 ################################################################################################
-if cat /etc/os-release | grep ^NAME | fgrep -q CentOS ; then
-    echo "You are running CentOS. Please also see Loris-MRI Readme for notes and links to further documentation in our main GitHub Wiki on how to install the DICOM Toolkit and other required dependencies."
-else
-    #Check if apt-get is installed
-    APTGETCHECK=`which apt-get`
-    if [ ! -f "$APTGETCHECK" ]; then
-        echo "\nERROR: Unable to find apt-get"
-        echo "Please ask your sysadmin or install apt-get\n"
-        exit
-    fi
-
-    echo "Installing DICOM Toolkit (May prompt for sudo password)"
-    sudo -S apt-get install dcmtk
+
+# Detecting distribution
+os_distro=$(hostnamectl |awk -F: '/Operating System:/{print $2}'|cut -f2 -d ' ')
+debian=("Debian" "Ubuntu")
```

Comment:
I get the following error here: 
```
Syntax error: "(" unexpected
```

Reply:
Hi Cecile, I ran the install script and I was unable to reproduce your error. I also don't see where there shouldn't be an '('. Would it be possible for you to double check or to give me more information about the error?